.travis.yml

```diff
@@ -6,6 +6,7 @@ node_js:
   - "4"
   - "6"
   - "7"
+  - "8"
```

Comment:
Given that Node.js 8 isn't *quite* LTS, let's also add it to the `allow_failures` matrix below.

Reply:
done